configs/detr/detr_r50_1x_coco.py

```diff
@@ -0,0 +1,94 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+model = dict(
+    type='DETR',
+    pretrained='torchvision://resnet50',
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=4,
+        out_indices=(3, ),
+        frozen_stages=1,
+        norm_cfg=dict(type='BN', requires_grad=True),
+        norm_eval=True,
+        style='pytorch'),
+    bbox_head=dict(
+        type='TransformerHead',
+        num_classes=80,
+        in_channels=2048,
+        num_fcs=2,
+        transformer=dict(
+            type='Transformer',
+            embed_dims=256,
+            num_heads=8,
+            num_encoder_layers=6,
+            num_decoder_layers=6,
+            feedforward_channels=2048,
+            dropout=0.1,
+            act_cfg=dict(type='ReLU', inplace=True),
+            norm_cfg=dict(type='LN'),
+            num_fcs=2,
+            pre_norm=False,
+            return_intermediate_dec=True),
+        position_encoding=dict(
+            type='SinePositionEmbedding', num_feats=128, normalize=True),
+        loss_cls=dict(
+            type='CrossEntropyLoss',
+            bg_cls_weight=0.1,
+            use_sigmoid=False,
+            loss_weight=1.0,
+            class_weight=1.0),
+        loss_bbox=dict(type='L1Loss', loss_weight=5.0),
+        loss_giou=dict(type='GIoULoss', loss_weight=2.0)))
+# training and testing settings
+train_cfg = dict(
+    _delete_=True,
+    assigner=dict(
+        type='HungarianMatcher', cls_weight=1., bbox_weight=5.,
+        giou_weight=2.),
+    pos_weight=-1)
+test_cfg = dict(_delete_=True, max_per_img=100)
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='Resize',
+        img_scale=[(480, 1333), (512, 1333), (544, 1333), (576, 1333),
+                   (608, 1333), (640, 1333), (672, 1333), (704, 1333),
+                   (736, 1333), (768, 1333), (800, 1333)],
+        multiscale_mode='value',
+        keep_ratio=True),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='Pad', size_divisor=1),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels'])
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=(1333, 800),
+        flip=False,
+        transforms=[
+            dict(type='Resize', keep_ratio=True),
+            dict(type='RandomFlip'),
+            dict(type='Normalize', **img_norm_cfg),
+            dict(type='Pad', size_divisor=1),
+            dict(type='ImageToTensor', keys=['img']),
+            dict(type='Collect', keys=['img'])
+        ])
+]
+data = dict(
+    samples_per_gpu=4,
```

Comment:
Since the `samples_per_gpu=4`, the config should be named as `detr_r50_8x4_1x_coco.py`. See config conventions [here](https://mmdetection.readthedocs.io/en/latest/config.html).